gradle.properties

```diff
@@ -4,7 +4,7 @@ org.gradle.jvmargs  = -Xmx3G
 org.gradle.daemon   = false
 
 # Core properties
-mod_version         = 1.2.0dev
+mod_version         = 1.2.7
```

Comment:
You don't need to change this, the build will change it automatically

Reply:
Oof ok.  😭 Is this something to make another PR for?

Reply:
you can just push a new commit to your branch and it should show up